doc/R.nvim.txt

```diff
@@ -103,28 +103,30 @@ Main dependencies:
       On Windows, you have to install Rtools to be able to build `nvimcom`:
       https://cran.r-project.org/bin/windows/Rtools/
 
-   Note about the R package `nvimcom`:
+      Note about the R package `nvimcom`:
 
-     - You do not need to load nvimcom in your .Rprofile because the R.nvim
-       plugin sets the environment variable `R_DEFAULT_PACKAGES`, including
-       `nvimcom` in the list of packages to be loaded on R startup. However,
-       if you set the option `defaultPackages` in a .Rprofile, you should
-       include "nvimcom" among the packages to be loaded (see
-       |nvimcom-not-loaded|).
+      You do not need to load nvimcom in your .Rprofile because the R.nvim
+      plugin sets the environment variable `R_DEFAULT_PACKAGES`, including
+      `nvimcom` in the list of packages to be loaded on R startup. However,
+      if you set the option `defaultPackages` in a .Rprofile, you should
+      include "nvimcom" among the packages to be loaded (see
+      |nvimcom-not-loaded|).
+
+   Quarto: required to complete chunk options in `#|` comments in Quarto, Rmd
```

Comment:
```suggestion
   Quarto: the `quarto` R package is required to complete chunk options in `#|` comments in Quarto, Rmd
```

Reply:
We need the completion data from the quarto _application_ for Rmd, Quarto, and Rnoweb. The quarto _package_ is required only to compile Quarto documents directly from R.

Reply:
ah ok from the CLI installation?

Reply:
Yes.